pkg/trisa/gds/models/v1beta1/models.go

```diff
@@ -20,22 +20,22 @@ const (
 
 // VASP Category Enumeration Helpers
 const (
-	VASPCategoryUnknown          = VASPCategory_UNKNOWN_VASP
-	VASPCategoryATM              = VASPCategory_ATM
-	VASPCategoryExchange         = VASPCategory_EXCHANGE
-	VASPCategoryHighRiskExchange = VASPCategory_HIGH_RISK_EXCHANGE
+	VASPCategoryUnknown    = "Unknown"
+	VASPCategoryExchange   = "Exchange"
+	VASPCategoryDEX        = "DEX"
+	VASPCategoryP2P        = "P2P"
+	VASPCategoryKiosk      = "Kiosk"
+	VASPCategoryCustodian  = "Custodian"
+	VASPCategoryOTC        = "OTC"
+	VASPCategoryFund       = "Fund"
+	VASPCategoryProject    = "Project"
+	VASPCategoryGambling   = "Gambling"
+	VASPCategoryMiner      = "Miner"
+	VASPCategoryMixer      = "Mixer"
+	VASPCategoryIndividual = "Individual"
+	VASPCategoryOther      = "Other"
```

Comment:
Primary update from Catherine et al: string VASP categories instead of an enum. These categories are from JJ.